decidim-elections/app/helpers/decidim/elections/votes_helper.rb

```diff
@@ -14,7 +14,9 @@ def ordered_answers(question)
       end
 
       def more_information?(answer)
-        answer.description || answer.proposals.any? || answer.photos.any?
+        translated_attribute(answer.description).present? ||
```

Comment:
Can you please add a spec in https://github.com/decidim/decidim/blob/develop/decidim-elections/spec/system/vote_online_spec.rb and https://github.com/decidim/decidim/blob/develop/decidim-elections/spec/helpers/decidim/elections/votes_helper_spec.rb? Thanks

Reply:
Done!